packages/shared/lib/chart.ts

```diff
@@ -31,9 +32,16 @@ interface ActivityTimeframe {
 }
 
 export enum DashboardChartType {
+    HOLDINGS = 'holdings',
     PORTFOLIO = 'portoflio',
     TOKEN = 'token',
 }
+
+export enum WalletChartType {
```

Comment:
Hmm is there a reason these are two separate `ChartType`s? It slightly confusing since `selectedWalletChart` and `selectedDashboardChart` use both the `DashboardChartType` type.

Reply:
Ooops, `selectedWalletChart` was supposed to use `WalletChartType` not `DashboardChartType`. Created a new enum because the two dashboards have different lists of charts that can be displayed.

Reply:
Updated to use the correct enum in the store.